frontend/src/components/icons/AddIcon.js

```diff
@@ -2,11 +2,11 @@ import React from "react";
 
 export default function AddIcon(){
     return(
-        <div>
-        <svg width="55" height="79" viewBox="0 0 55 79" fill="none" xmlns="http://www.w3.org/2000/svg">
-            <path d="M27.5 0.75C12.5038 0.75 0.375 12.8788 0.375 27.875C0.375 48.2188 27.5 78.25 27.5 78.25C27.5 78.25 54.625 48.2188 54.625 27.875C54.625 12.8788 42.4962 0.75 27.5 0.75Z" fill="#FF0046"/>
-            <path fill-rule="evenodd" clip-rule="evenodd" d="M29.1739 16.5C29.1739 15.6716 28.5023 15 27.6739 15C26.8455 15 26.1739 15.6716 26.1739 16.5V27.1739L15.5 27.1739C14.6716 27.1739 14 27.8455 14 28.6739C14 29.5023 14.6716 30.1739 15.5 30.1739L26.1739 30.1739V41.5C26.1739 42.3284 26.8455 43 27.6739 43C28.5023 43 29.1739 42.3284 29.1739 41.5V30.1739L40.5 30.1739C41.3284 30.1739 42 29.5023 42 28.6739C42 27.8455 41.3284 27.1739 40.5 27.1739L29.1739 27.1739V16.5Z" fill="white"/>
-        </svg>
-            </div>
+        <a href="/new">
+            <svg href="/new" width="55" height="79" viewBox="0 0 55 79" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path d="M27.5 0.75C12.5038 0.75 0.375 12.8788 0.375 27.875C0.375 48.2188 27.5 78.25 27.5 78.25C27.5 78.25 54.625 48.2188 54.625 27.875C54.625 12.8788 42.4962 0.75 27.5 0.75Z" fill="#FF0046"/>
+                <path fill-rule="evenodd" clip-rule="evenodd" d="M29.1739 16.5C29.1739 15.6716 28.5023 15 27.6739 15C26.8455 15 26.1739 15.6716 26.1739 16.5V27.1739L15.5 27.1739C14.6716 27.1739 14 27.8455 14 28.6739C14 29.5023 14.6716 30.1739 15.5 30.1739L26.1739 30.1739V41.5C26.1739 42.3284 26.8455 43 27.6739 43C28.5023 43 29.1739 42.3284 29.1739 41.5V30.1739L40.5 30.1739C41.3284 30.1739 42 29.5023 42 28.6739C42 27.8455 41.3284 27.1739 40.5 27.1739L29.1739 27.1739V16.5Z" fill="white"/>
+            </svg>
+        </a>
```

Comment:
Just because I am interested. What do the individual numbers stand for?

Reply:
I didn't quite understand, but this was exported from Figma. Generally you can convert SVGs to code which look like line 7 in their most basic form.